src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -344,6 +345,10 @@ public function getEnv(string $prefix, string $name, \Closure $getEnv): mixed
             return trim($env);
         }
 
+        if ('urlencode' === $prefix) {
+            return urlencode($env);
```

Comment:
`rawurlencode` should be used instead. It encodes spaces as `%20` while `rawurlencode` encode them as `+`.

Reply:
Ive replaced it.